docs/docsite/rst/porting_guides/porting_guide_2.8.rst

```diff
@@ -110,6 +110,10 @@ Noteworthy module changes
 
 * The ``vmware_local_role_facts`` module now returns a list of dicts instead of a dict of dicts for role information.
 
+* The ``vmware_local_role_facts`` module now returns a list of dicts instead of a dict of dicts for role information.
```

Comment:
I think this line is extra.

Reply:
Whoops, correct. That shouldn't have been there. 

Removed.